pages/common/pueue-send.md

```diff
@@ -0,0 +1,16 @@
+# pueue send
+
+> Send input to a task.
+> More information: <https://github.com/Nukesor/pueue>.
+
+- Send input to a running command:
+
+`pueue send {{task_id}} "{{input}}"`
+
+- Send confirmation to a task expecting y/N (e.g. apt, cp):
```

Comment:
Though I agree the intention with such an example, I don't think it's that practical, actually, because it's just a copy of the first example, and users are likely to pass the equivelent `-y` flag to the command in question to prevent something like this from being needed.  Therefore, I think we should remove this example.